cloudsync/tests/test_cs.py

```diff
@@ -99,6 +99,7 @@ def fixture_multi_local_cs(mock_provider_generator):
     yield from multi_local_cs_generator(2, mock_provider_generator)
```

Comment:
what is the "onedrive" submodule?

Reply:
this was an experiment for a local dev environment, not going to be submitted.